pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -327,6 +329,15 @@ func managePod_v311_00_to_latest(ctx context.Context, configMapsGetter corev1cli
 		required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, "--tls-private-key-file=/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.key")
 	}
 
+	config, err := configInformer.Config().V1().Schedulers().Lister().Get("cluster")
+	if err != nil {
+		return nil, false, err
+	}
+	if len(config.Spec.Policy.Name) > 0 {
+		required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, "--policy-configmap=policy-configmap")
```

Comment:
`required` is always a fresh pod manifest so there should be no argument conflicts